kolibri_instant_schools_plugin/assets/src/views/about/state/actions.js

```diff
@@ -0,0 +1,23 @@
+import { PageNames } from '../constants';
+import { createTranslator } from 'kolibri.utils.i18n';
+
+const translator = createTranslator('aboutPageTitles', {
+  aboutPageTitle: 'About',
+  faqPageTitle: 'FAQ',
+});
+
+function showAbout(store) {
+  store.dispatch('SET_PAGE_NAME', PageNames.ABOUT);
+  store.dispatch('CORE_SET_TITLE', translator.$tr('aboutPageTitle'));
+  store.dispatch('CORE_SET_ERROR', null);
+  store.dispatch('CORE_SET_PAGE_LOADING', false);
```

Comment:
is this mutation necessary? usually this is mutation is used to close out a `dispatch('CORE_SET_PAGE_LOADING', true)` that is done at the begin of an asynchronous action.

Reply:
Not necessary.

Reply:
Did need it after all, since the default value for `loading` within the core state, is `true`.